components/genesys-email-flow/modules/email_flow/main.tf

```diff
@@ -0,0 +1,31 @@
+
+terraform {
+  required_providers {
+    genesyscloud = {
+      source = "mypurecloud/genesyscloud"
+
+    }
+  }
+}
+
+#
+# Description:  
+#
+# There is some hackery a foot here. Archy is a standalone command line tool so it can not participate in all of the terraform goodness natively.
+# Using a local-exec to handle the creation and deletion of an archy flow.  Note:  I can not seem to delete a flow with the archy tool so I use
+# a shell script to call the Genesys Cloud CLI to lookup the flow and then delete it via the CLI
+# 
+
+resource "null_resource" "deploy_files" {
+
+
+  provisioner "local-exec" {
+    command = "${path.module}/scripts/manage_flow.py CREATE ${var.genesys_email_domain} ${var.genesys_email_domain_region}"
```

Comment:
Also line 28. Tested this in both Windows and Linux, but .py files don't automatically get run as Python scripts by default. Prepended python (python3 for Linux) to both commands so it would be executed.

Reply:
Fixed.  I also made a comment in the code.  I was relying on a #!/usr/local/bin/python3 in the code